lms/templates/courseware/course_about.html

```diff
@@ -319,6 +319,14 @@
             </li>
           % endif
 
+          % if pre_requisite_course:
+          <li class="prerequisite-course important-dates-item">
+            <i class="icon icon-list-ul"></i>
+            <p class="important-dates-item-title">${_("Requirements")}</p>
```

Comment:
@ziafazal I'm only seeing a **Requirements** section, and it contains what we're calling a prerequisite - ?
![screenshot 228](https://cloud.githubusercontent.com/assets/3493253/5647162/646991be-9651-11e4-9ac0-2f5a7834e241.png)